uploadToNexus.xml

```diff
@@ -327,18 +309,18 @@
         <!--   canonical modelgen processor during compilation. It has no real dependencies, nor is anything really       -->
         <!--   dependant upon it, but it is needed for this compiler functionality to work. (think of it as a compiler    -->
         <!--   add-on). It will need to be downloaded independently for this feature (modelgen) to work.                  -->
-        <property name="eclipselink.dependencies" value="${deps.head}${dep.persistence}${dep.sdoapi}${dep.validation}${dep.json}${deps.foot}"/>
+        <property name="eclipselink.dependencies" value="${deps.head}${dep.persistence}${dep.sdoapi}${deps.foot}"/>
         <!-- would need the modelgen.jar for modelgen but cannot include dep -->
         <property name="jpql.dependencies"        value="${deps.head}${dep.hermes}${deps.foot}"/>
-        <property name="core.dependencies"        value="${deps.head}${dep.asm}${dep.json}${deps.foot}"/>
```

Comment:
Core still depends on javax.json for compile, yes?

Reply:
Yes and that dependency comes from manifest in the core project (where it is optional), not from pom.xml. This pom and the pom you see in the project sources are different and used for different things - this one is for artifacts in maven repo (see the building el wiki for how to check this locally) and the latter is for "tycho magic"